docs/.sphinx/build_requirements.py

```diff
@@ -0,0 +1,127 @@
+import sys
```

Comment:
I'm not sure if this file is meant to be checked in.

Let's go though entire `.sphinx` and `_templates`... maybe these are automatically pulled in at build time, in which case they should not be checked in.

Reply:
This file is used to regenerate the updated `requirements.txt` file. If we don't have it in our repo, we can also clone the starter pack repo, run the file there, and copy the generated `requirements.txt` to our repo, it is slightly a bit more manual work.

Since we have this file in the operator repo, for consistency, let's keep it here?

Reply:
For now, I will keep this file and resolve this comment, since it seems @AnneCYH agreed to do so.